### DISCUSSION TITLE
Error message: legacy editor (not using it though!)

### DISCUSSION THREAD

**Discussion by gathering-moss** [vslinko/obsidian-outliner#338] [Q&A] [answered]
I keep getting this error messages when I start Obsidian, but I am not using the legacy editor. Anyone else having this problem and/or know now to fix it?

> Outliner plugin does not support legacy editor mode stating from version 2.0. Please disable the "Use legacy editor" option or manually install version 1.0 of Outliner plugin.

I’m running Obsidian v0.15.8 (installer version: v0.14.15). I have the latest plugin version downloaded as well (Outliner 2.3.0).

For what it's worth, I am getting the same error message for the Zoom plugin.

**Comment by gathering-moss** [comment, marked as answer]
I was able to figure out what happened. It was a sync issue. A computer I don't use often still had the legacy editor toggled on, so I turned it off. Everything synced and all is well.

**Comment by antiduplicity** [comment]
I have never turned on legacy editor and never will and I am told to not turn it on every time I open obsidian.  So let's get rid of this error message place
